<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValveClipDeviceSimulatorWidget</class>
 <widget class="QWidget" name="ValveClipDeviceSimulatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ValveClipDeviceSimulatorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1" colspan="2">
    <widget class="qSlicerMarkupsPlaceWidget" name="guideCurvePlaceWidget">
     <property name="toolTip">
      <string>Define guide semi-rigid tube (8.1mm outer diameter)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Guide curve: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="curvatureFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Curvature calculation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="curvatureCalculationCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="curvatureValuesFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Guide:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="guideCurvatureMeanTextLabel">
           <property name="text">
            <string>mean: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="catheterCurvatureMeanValueLabel">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="guideCurvatureMeanValueLabel">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Catheter:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="catheterCurvatureMeanTextLabel">
           <property name="text">
            <string>mean: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="curvatureColorInterpolationCheckbox">
           <property name="toolTip">
            <string>If on, then the green-yellow-red colors are interpolated for in-between curvature values.
Otherwise only show the discrete colors (hard stop)</string>
           </property>
           <property name="text">
            <string>Color interpolation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="guideCurvatureMaxTextLabel">
           <property name="text">
            <string>  max: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="guideCurvatureMaxValueLabel">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="catheterCurvatureMaxTextLabel">
           <property name="text">
            <string>  max: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="catheterCurvatureMaxValueLabel">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerMarkupsPlaceWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerMarkupsPlaceWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
